CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
 # Release History
-## 0.10.0 (Unreleased)
+## 0.10.0 (2022-09-16)
 
 ### New Features:
+- Added support for evaluating Snowpark dataframes asynchronously :
+  - Added keyword argument `block` to most action APIs on Snowpark dataframes (which executes queries) to allow asynchronous evaluations.
+  - Added method `df.collect_nowait()` to asynchronous evaluations.
+  - Added class `AsyncJob` to retrieve results from asynchronously executed queries and check their statuses.
+  - This feature is experimental and APIs are subject to change.
```

Comment:
These are children items of the 1st statement. How about indenting them to the right?

Reply:
My bad. It's already indented.